crates/fj-proc/src/attributed_arguments.rs

```diff
@@ -0,0 +1,198 @@
+use proc_macro::TokenStream;
+use quote::quote;
+use syn::{
+    bracketed, parenthesized, parse::Parse, parse_macro_input, parse_quote,
+};
+
+pub fn attributed_arguments(_: TokenStream, input: TokenStream) -> TokenStream {
```

Comment:
I think having two different attributes (and two different models for testing/demonstrating) them is overkill. I like the approach implemented in this one much better. It would be better, if this just replaced the other one.

Reply:
The first one was more or less just left in because it was the first one I came up with. So more for completeness than anything else. I'll get rid of it :)